cc/tflite_micro/tflite_micro.h

```diff
@@ -27,7 +27,7 @@ extern "C" {
 int tflite_init(const uint8_t* model_bytes, size_t model_bytes_len,
                 const uint8_t* tensor_arena_bytes, size_t tensor_arena_bytes_len);
 
-int tflite_run(const uint8_t* input_bytes, size_t input_bytes_len, uint8_t* output_bytes,
+int tflite_run(const uint8_t* input_bytes, size_t input_bytes_len, uint8_t** output_bytes,
```

Comment:
I think the out param should be called `output_bytes_ptr` or something like that? Also it needs some documentation if nothing else to explain who owns that memory after calling this function.